packages/aws-cdk/lib/api/util/cloudformation/stack-activity-monitor.ts

```diff
@@ -354,7 +354,7 @@ abstract class ActivityPrinterBase implements IActivityPrinter {
       this.resourcesInProgress[activity.event.LogicalResourceId] = activity;
     }
 
-    if (status.endsWith('_FAILED')) {
+    if (hasErrorMessage(status)) {
```

Comment:
This is not related, right?

Reply:
It is, it's to make the "failed assertion" messages properly show up in the stack monitor. Otherwise the only thing you see is `ROLLBACK_COMPLETE` which is not very helpful.

Reply:
Cool beans